nights/2023-07-01-README.md

```diff
@@ -1,35 +1,36 @@
 # 2023-07-01
 
-## Reviewed by:   your_name_here
+## Reviewed by:   Tanner Weyer
 
 ## Raw image gallery: http://physics.mnstate.edu/feder_gallery/2023-07-01
 
 ## Unusual images?
 
 Are there any images that look unusual? List the file name of any unusual images for this night here, with description:
 
-+ `crazy-object-001R.fit` -- not sure what this is an image of
-+ `another-crazy-object-003B.fit` -- this looks more like a flat than a science image.
-+ `full-moon-005B.fit` -- nice satellite or plane track in here.
++ `M101-S001-R001-C001-rp.fit` -- Target Object off center and high cloud
++ `M101-S001-R001-C002-rp.fit` -- Target Object off center and high cloud
++ `M101-S001-R001-C003-rp.fit` -- Target Object off center and high cloud
++ `M101-S001-R001-C004-rp.fit` -- Target Object off center and high cloud
++ `M101-S001-R001-C005-rp.fit` -- Target Object off center and high cloud
++ `M101-S001-R001-C007-rp.fit` -- Target Object off center and high cloud
++ `M101-S001-R001-C008-rp.fit` -- Target Object off center and high cloud
++ `M101-S001-R001-C009-rp.fit` -- Target Object off center and high cloud
++ `M101-S001-R001-C010-rp.fit` -- Target Object off center and high cloud
++ No flat images due to the run not completing fully. 
 
-Delete this list if there are no unusual images.
 
 ## Missing information?
 
 Check these off if they are true:
 
-- [ ] No images are missing filter information (except BIAS and DARK, which need no filter).
-- [ ] No images are missing pointing information (RA/Dec and WCS)
-- [ ] No images are missing object names (only applies to science images)
-- [x] EXAMPLE checked-off box, please delete.
+- [X] No images are missing filter information (except BIAS and DARK, which need no filter).
+- [X] No images are missing pointing information (RA/Dec and WCS)
+- [X] No images are missing object names (only applies to science images)
 
 If any images are missing information and you have been unable to fix them please list
 them below with a short description of the problem.
```

Comment:
```suggestion
them below with a short description of the problem.

The images below are missing astrometry because there are too many clouds.

+ `M101-S001-R001-C002-rp.fit`
+ `M101-S001-R001-C003-rp.fit`
+ `M101-S001-R001-C004-rp.fit`

```